src/Form/Control/ScopeBuilder.php

```diff
@@ -218,7 +218,7 @@ class ScopeBuilder extends Control
         'enum' => [
             'type' => 'select',
             'operators' => [
-                self::OPERATOR_EQUALS,
+                self::OPERATOR_TEXT_EQUALS,
```

Comment:
what is the difference and why the condition below is not updated as well?

Reply:
Was not sure myself, so this is why I asked @georgehristov to take a look.

Reply:
It does fix the issue but we may have to look further.